wagtail/snippets/views/snippets.py

```diff
@@ -1346,7 +1357,11 @@ def get_inspect_template(self):
         )
 
     def get_admin_url_namespace(self):
-        """Returns the URL namespace for the admin URLs for this model."""
+        """
+        Returns the URL namespace for the admin URLs for this model.
+
+        **Deprecated** - the preferred way to customise this is to define a ``url_namespace`` property.
```

Comment:
Good question! My inclination would be to leave it as a convenience, since we've only just persuaded people to move their ModelAdmin definitions over, and having to do further seemingly-minor tweaks as a follow-up might annoy people. I don't feel particularly strongly either way though - if it makes the SnippetViewSet code cleaner I'm happy to go ahead.